presto-record-decoder/src/main/java/io/prestosql/decoder/DecoderModule.java

```diff
@@ -43,7 +41,6 @@ public void configure(Binder binder)
         decoderFactoriesByName.addBinding(CsvRowDecoder.NAME).to(CsvRowDecoderFactory.class).in(SINGLETON);
         decoderFactoriesByName.addBinding(JsonRowDecoder.NAME).to(JsonRowDecoderFactory.class).in(SINGLETON);
         decoderFactoriesByName.addBinding(RawRowDecoder.NAME).to(RawRowDecoderFactory.class).in(SINGLETON);
-        decoderFactoriesByName.addBinding(AvroRowDecoder.NAME).to(AvroRowDecoderFactory.class).in(SINGLETON);
```

Comment:
I don't like it. I haven't noticed that earlier, I am sorry about it.

Previously `DecoderModule` was self contained. Now it has a dependency that you need to provide externally `AvroModule`. That could be non intuitive. Maybe let's make `DecoderModule` constructor to accept a module that is expected to provide Avro decoders, then default constructor just use `AvroDecoderModule`. WDYT? Then you don't need to change other connectors.

Reply:
Sounds good!